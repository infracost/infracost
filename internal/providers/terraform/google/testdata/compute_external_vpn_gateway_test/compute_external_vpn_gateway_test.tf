provider "google" {
  credentials = "{\"type\":\"service_account\"}"
  region      = "us-central1"
}

resource "google_compute_external_vpn_gateway" "my_compute_external_vpn_gateway" {
  name            = "my_compute_external_vpn_gateway"
  redundancy_type = "SINGLE_IP_INTERNALLY_REDUNDANT"
  description     = "An externally managed VPN gateway"
  interface {
    id         = 0
    ip_address = "8.8.8.8"
  }
}
