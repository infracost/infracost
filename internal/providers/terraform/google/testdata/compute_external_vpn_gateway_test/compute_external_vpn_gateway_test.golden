
 Name                                                                                             Monthly Qty  Unit  Monthly Cost 
                                                                                                                                  
 google_compute_external_vpn_gateway.my_compute_external_vpn_gateway                                                              
 └─ Network egress                                                                                                                
    ├─ IPSec traffic to worldwide excluding China, Australia but including Hong Kong (first 1TB)        1,024  GB         $122.88 
    ├─ IPSec traffic to worldwide excluding China, Australia but including Hong Kong (next 9TB)         9,216  GB       $1,013.76 
    ├─ IPSec traffic to worldwide excluding China, Australia but including Hong Kong (over 10TB)        2,260  GB         $180.80 
    ├─ IPSec traffic to China excluding Hong Kong (first 1TB)                                           1,024  GB         $235.52 
    ├─ IPSec traffic to China excluding Hong Kong (next 9TB)                                            7,476  GB       $1,644.72 
    └─ IPSec traffic to Australia (first 1TB)                                                             250  GB          $47.50 
                                                                                                                                  
 OVERALL TOTAL                                                                                                          $3,245.18 